File: frontend/src/app/utils/review.ts
import { CustomUser } from "./user";
import { Book } from "./book";

export class Review {
    id: number;
    user: CustomUser;
    book: Book;
    review: string;
  
    constructor(id: number, user: CustomUser, book: Book, review: string) {
      this.id = id;
      this.user = user;
      this.book = book;
      this.review = review;
    }
  }

File: views_antigas/review.py
from django_filters.rest_framework import DjangoFilterBackend
from rest_framework import viewsets
from ..models import Review
from ..serializers import ReviewSerializer

class ReviewViewSet(viewsets.ModelViewSet):
    queryset = Review.objects.all()
    serializer_class = ReviewSerializer
    filter_backends = [DjangoFilterBackend]
    filterset_fields = ['book', 'user']